lr_face/models.py

```diff
@@ -1,14 +1,23 @@
 from __future__ import annotations
 
+import hashlib
 import importlib
+import os
+import pickle
 from enum import Enum
-from typing import Tuple, List
+from typing import Tuple, List, Optional, Union
 
 import numpy as np
 import tensorflow as tf
 from scipy import spatial
 
+from lr_face.data import FaceImage, FaceTriplet, to_array, FacePair
+from lr_face.losses import TripletLoss
 from lr_face.utils import cache
+from lr_face.versioning import Version
+
+EMBEDDINGS_DIR = 'embeddings'
+MODELS_DIR = 'models'
```

Comment:
'weights' is mssn duidelijker?

Reply:
Zal ik aanpassen

- [ ] hernoem `MODELS_DIR` van 'models' naar 'weights'